geozero/src/lib.rs

```diff
@@ -25,6 +25,7 @@
 //! | GeoJSON   | [GeoJson](geojson::GeoJson), [GeoJsonString](geojson::GeoJsonString) | XYZ        | [GeoJsonReader](geojson::GeoJsonReader), [GeoJson](geojson::GeoJson) | [ToJson]            | [GeoJsonWriter](geojson::GeoJsonWriter) |
 //! | GDAL      | `gdal::vector::Geometry`                                             | XYZ        | -                                                                    | [ToGdal]            | [GdalWriter](gdal::GdalWriter)          |
 //! | GEOS      | `geos::Geometry`                                                     | XYZ        | -                                                                    | [ToGeos]            | [GeosWriter](geos::GeosWriter)          |
+//! | GPX       |                                                                      | XY         |                                                                      |                     |                                         |
```

Comment:
What goes here?

Reply:
Would be the structs implementing `GeozeroGeometry`,  `GeomProcessor`, etc.

Reply:
I didn't add a new struct in this pull request. Should I?

Reply:
You don't have to, but if you add a `GpxReader` struct implementing `GeozeroDatasource` (like e.g. `GeoJsonReader`), users can do e.g.
`let wkt = GpxReader(&mut f).to_wkt()?;`